Tools/WinMLRunner/CommandLineArgs.h

```diff
@@ -90,4 +95,7 @@ class CommandLineArgs
     std::wstring m_inputData;
     std::wstring m_outputPath;
     uint32_t m_numIterations = 1;
+
+    float m_scaleFactor = 1.0f;
+    std::array<float, 3> m_meanStdDev{ 0, 0, 0 };
```

Comment:
Please rename or add a comment to indicate that m_meanStdDev corresponds to RBG, BGR, or whatever pixel format.

Alternatively a struct may be more informative here.